c_src/get_version.sh

```diff
@@ -1,4 +1,3 @@
-#!/bin/sh
+#!/usr/bin/env bash
 
-
-grep '.*define.*(.*VERSION' "$1"/src/jq.erl | sed -e 's/.*VERSION.*,.*\([[:digit:]]\+\).*/\1/'
+grep '.*define.*(.*VERSION' "$1"/src/jq.erl | sed -E 's/.*VERSION.*,.*([[:digit:]]+).*/\1/'
```

Comment:
Does this script depend on any bash feature? If not, it seems unnecessary to add bash as a dependency.

Reply:
I'll change it back to sh.